firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nBaseSnapshotTests.swift

```diff
@@ -126,3 +126,9 @@ class L10nBaseSnapshotTests: XCTestCase {
         mozWaitForElementToNotExist(progressIndicator, timeout: 30.0)
     }
 }
+
+extension XCUIElement {
+    func tapOnApp() {
```

Comment:
There is a new compilation error on line 372 of FxScreenGraph.swift. Without this `extension`, I can't compile L10nSnapshotTests successfully.